src/meta/sql.rs

```diff
@@ -877,17 +877,13 @@ mod tests {
             ("select * from tbl where b = 2 AND (_timestamp BETWEEN 1666093521151350 AND 1666093521151351)", (1666093521151350,1666093521151351)),
             ("select * from tbl where (_timestamp BETWEEN 1666093521151350 AND 1666093521151351)", (1666093521151350,1666093521151351)),
             ("select * from tbl where _timestamp BETWEEN 1666093521151350 AND 1666093521151351", (1666093521151350,1666093521151351)),
-            ("select * from tbl where time_range(\"_timestamp\", '2022-10-19T15:19:24.587Z','2022-10-19T15:34:24.587Z')",(1666192764587000,1666193664587000))].to_vec();
-        for sql in sqls {
-            let meta = Sql::new(sql.0);
-            if meta.is_err() {
-                println!("sql: {}, err: {}", sql.0, meta.as_ref().err().unwrap());
-            }
-            assert!(meta.is_ok());
-            let time_range = meta.unwrap().time_range.unwrap();
-            assert_eq!(time_range.0, sql.1 .0);
-            if sql.1 .1 != 0 {
-                assert_eq!(time_range.1, sql.1 .1);
+            (r#"select * from tbl where time_range("_timestamp", '2022-10-19T15:19:24.587Z','2022-10-19T15:34:24.587Z')"#,(1666192764587000,1666193664587000))].to_vec();
+
+        for (sql, (expected_t1, expected_t2)) in samples {
+            let (actual_t1, actual_t2) = Sql::new(sql).unwrap().time_range.unwrap();
```

Comment:
Same, it's shortly, but we can't directly use unwrap the SQL maybe error.

Reply:
If `Sql::new` returns `Err`, we want the test to panic. This code is equivalent to the original, more roundabout, implementation.

https://github.com/zinclabs/zincobserve/blob/14ea6001d6f4229dd49fb53a8affc18962133241/src/meta/sql.rs#L882-L886

BTW, if we want to see `sql` in case of error, we should replace `Sql::new(sql).unwrap()` with `Sql::new(sql).expect("sql={sql}")` or something like this.